locust/stats.py

```diff
@@ -454,12 +463,33 @@ def print_stats(stats):
     total_rps = 0
     total_reqs = 0
     total_failures = 0
+    total_min = None
+    running_avg = 0
+    running_median = []
+    total_max = 0
+    total_stat_methods = len(stats)
+
     for key in sorted(stats.iterkeys()):
         r = stats[key]
         total_rps += r.current_rps
         total_reqs += r.num_requests
         total_failures += r.num_failures
+        running_avg += r.avg_response_time
+        running_median.append(r.median_response_time)
+
+        if total_min is None:
+            total_min = r.min_response_time
+
+        if total_max < r.max_response_time:
+            total_max = r.max_response_time
+
+        if total_min > r.min_response_time:
+            total_min = r.min_response_time
+
         console_logger.info(r)
+
+    total_average = float(running_avg / total_stat_methods)
```

Comment:
If locust is exited before executing any requests, this line will throw a ZeroDivisionError.